migrations/versions/fd467d5e0812_.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""empty message
+
+Revision ID: fd467d5e0812
+Revises: 19dd4e2c1e63
+Create Date: 2022-04-06 10:25:46.398750
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+import app
+import app.extensions
+
+
+# revision identifiers, used by Alembic.
+revision = 'fd467d5e0812'
+down_revision = '19dd4e2c1e63'
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('asset_group_sighting', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('detection_attempts', sa.Integer(), nullable=False))
```

Comment:
having a default value on the column `0` is the only suggestion i have here.  save broken migrations on existing data in `asset_group_sighting` instances.

Reply:
Agreed, done, with a quoted value which feels utterly wrong